src/components/SettlementButton.js

```diff
@@ -123,11 +123,11 @@ class SettlementButton extends React.Component {
                 paymentMethod = CONST.IOU.PAYMENT_TYPE.ELSEWHERE;
             }
 
-            // No options are shown only in case of the report preview settlement button where we want to use same short button label for all the options
+            // In case of the settlement button in the report preview component, we do not show payment options and the label for Wallet and ACH type is simply "Pay".
             return [
```

Comment:
NAB: 

// When displayed within the report preview component, we should not show payment options and the label for Wallet and ACH type is simply "Pay".

Reply:
Suggesting because this was a bit confusing to me.